frontend/src/cards/ui/SviAlert.tsx

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import Alert from "@material-ui/lab/Alert";
+import variables from "../../styles/variables.module.scss";
+import { MetricQueryResponse } from "../../data/query/MetricQuery";
+import { Fips } from "../../data/utils/Fips";
+
+interface SviAlertProps {
+  svi: number;
+  sviQueryResponse: MetricQueryResponse;
+  fips: Fips;
+}
+
+const findRating = (svi: number) => {
+  if (svi < 0.34) {
+    return "low";
+  }
+  if (svi > 0.67) {
+    return "high";
+  } else return "medium";
+};
+
+const findColor = (rating: string) => {
+  if (rating === "high") {
+    return "#d32f2f";
+  }
+  if (rating === "low") {
+    return variables.altGreen;
+  } else return "#d85c47";
+};
+
+function SviAlert(props: SviAlertProps) {
+  const rating = findRating(props.svi);
+  const color = findColor(rating);
+
+  return (
+    <>
+      {props.svi > 0 ? (
+        <Alert severity="info" style={{ margin: "10px" }}>
+          This county has a social vulnerability index of <b>{props.svi}</b>;
+          which indicates a{" "}
+          <a href="testing" style={{ textDecorationColor: color }}>
+            <span style={{ color: color, fontWeight: "bold" }}>
```

Comment:
I know there are haters out there, but we generally just use `<b>` tags for bold text. Perhaps down the line we should create a global class to handle emphasized text, but for now let's just stay consistent with `<b>`. In general I think we should avoid inline CSS as well, except rare places where it's needed